source/release-notes/1.3.txt

```diff
@@ -9,26 +9,41 @@ Release Notes for ``mongosync`` 1.3
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
 .. note::
 
    Upcoming.
 
+New Features
+-------------
 
 Capped Collections
-------------------
+~~~~~~~~~~~~~~~~~~
 
 Starting in 1.3.0, {+c2c-product-name+} supports :ref:`capped
 collections <manual-capped-collection>`. For more information, see
 :ref:`c2c-capped-collections`.
 
 Disable Index Builds
---------------------
+~~~~~~~~~~~~~~~~~~~~
 
 Starting in 1.3.0, {+c2c-product-name+} supports disabling non-essential
 index builds on the destination cluster.  This can improve migration 
 performance, especially with index heavy workloads.
 
 For more information, see :ref:`c2c-api-start`.
+
+Progress Output
+~~~~~~~~~~~~~~~
+
+Starting in 1.3.0, {+c2c-product-name+} includes the ``mongosyncID`` and
+``coordinatorID`` in the :ref:`c2c-api-progress` response documents.  This
+helps determine whether the ``mongosync`` instance is serving as a coordinator.
+
+Bug Fixes
+----------
+
+* Sync cluster collections on replica sets fail to copy all documents to the
+  destination cluster (:issue:`REP-1479`)
```

Comment:
Isn't the issue (REP-1479) internal?

Reply:
@rohshar, Not sure off hand.  I can cut the link if you don't want it there.

Reply:
Yeah I would just rephrase the whole statement to say:

"Fixes a bug that could cause ``mongosync`` to not copy some documents to the destination cluster for clustered collections on replica sets."

Reply:
Fixed